elastic/kibana
# [Uptime] Updates default monitor management browser schedule to 10 minutes, and adjusts beta badge content

## Summary

Fixes https://github.com/elastic/synthetics-service/issues/409
--
Adjusts default schedule for browser monitors to 10 minutes.
<img width="528" alt="Screen Shot 2022-04-12 at 11 53 52 AM" src="https://user-images.githubusercontent.com/11356435/163010715-898d6b87-1907-4278-b1d2-05618c6899be.png">

Leaves default schedule for lightweight checks at 3 minutes.
<img width="530" alt="Screen Shot 2022-04-12 at 11 53 46 AM" src="https://user-images.githubusercontent.com/11356435/163010716-ae630a69-6585-44b3-9612-f715ae80cf48.png">

Testing
1. Navigate to monitor management. Click Add monitor (You may need to first enable monitor management).
2. Select HTTP, TCP, or icmp monitor type. Confirm the default schedule is 3 minutes
3. Select browser monitor type. Confirm the default schedule is 10 minutes

Fixes https://github.com/elastic/uptime/issues/470
--
Adjusts the content for the beta badge tooltip
<img width="1448" alt="Screen Shot 2022-04-12 at 12 12 02 PM" src="https://user-images.githubusercontent.com/11356435/163010814-dd3efd8d-cfef-43e0-b414-2c2f253a4792.png">

Testing
Navigate to monitor management. Find the monitor management badge next to the Monitor Management title. Hover over the badge to view the tooltip content
